### PR TITLE
Some little improvements

### PR DESCRIPTION
**Pull request by sbrunner**
* Be able to configure the database in one shot.
* Be able to configure the port as an environment variable.
* Use a correct value in the checker_url.
* Add changelog message about pg_trgm.
* Add commands to clean unused Docker artifacts.
* Remove relay old `create_db` script.

### CODE REVIEW

File: doc/integrator/docker.rst [hunk 1]
```diff
@@ -231,3 +231,21 @@ add something like that in your composition:
 
 When you do an upgrade, backup your DB and upgrade this container first. It will update your
 DB schema, if needed.
+
+Clean
+-----
+
+Docker don't clean anything automatically, and espacially the images take many place.
```
Comment (jwkaltz) [inline doc/integrator/docker.rst]: Suggestion:
Docker does not clean anything automatically, in particular it does not clean any images, therefore disk space may become problematic after a certain number of builds.

File: doc/integrator/docker.rst [hunk 2]
```diff
@@ -231,3 +231,21 @@ add something like that in your composition:
...
+You can use the following commands to do a maual clean:
```
Comment (jwkaltz) [inline doc/integrator/docker.rst]: Suggestion:
You can use the following commands to manually remove docker files.

File: doc/integrator/docker.rst [hunk 3]
```diff
@@ -231,3 +231,21 @@ add something like that in your composition:
...
+
+.. prompt:: bash
+
+   docker ps --all --quiet --filter status=exited | xargs --no-run-if-empty docker rm
+   docker images | grep "<none>" | awk '{print $3}' | xargs --no-run-if-empty docker rmi || true
+   docker volume ls --quiet --filter dangling=true | grep '[0-9a-f]\{64\}' | xargs --no-run-if-empty docker volume rm
+
+It will removed::
```
Comment (jwkaltz) [inline doc/integrator/docker.rst]: This will remove

File: doc/integrator/docker.rst [hunk 4]
```diff
@@ -231,3 +231,21 @@ add something like that in your composition:
...
+
+  * Container with exit status
+  * Images with version on name as `<none>`
+  * Unnamed dagling volumes
```
Comment (jwkaltz) [inline doc/integrator/docker.rst]: dangling

**Comment by sbrunner** [comment]
Thanks @jwkaltz

**Pull request merged**